pkg/securitypolicy/securitypolicy.go

```diff
@@ -238,13 +241,14 @@ func CreateContainerPolicy(
 }
 
 // NewSecurityPolicy creates a new SecurityPolicy from the provided values.
-func NewSecurityPolicy(allowAll bool, containers []*Container) *SecurityPolicy {
+func NewSecurityPolicy(allowAll bool, logging bool, containers []*Container) *SecurityPolicy {
```

Comment:
as much as a i hate the "options pattern" in Go, given we use it elsewhere, shouldnt we use it here for allowAll and logging?

I dont think this variable should be `logging` that says to me, "is this going to log", not "is logging allowed".

Reply:
sure

Reply:
is there a downside to being consistent like that? would it make NewSecurityPolicy partial and require error checking? If yes, then I think what you have is far superior to the other pattern (which I'm already not a fan of).

Reply:
yeah, I'll leave it as is it is for now.